docs/docs.go

```diff
@@ -0,0 +1,66 @@
+// Package docs GENERATED BY SWAG; DO NOT EDIT
+// This file was generated by swaggo/swag
+package docs
+
+import "github.com/swaggo/swag"
+
+const docTemplate = `{
+    "schemes": {{ marshal .Schemes }},
+    "swagger": "2.0",
+    "info": {
+        "description": "{{escape .Description}}",
+        "title": "{{.Title}}",
+        "contact": {},
+        "version": "{{.Version}}"
+    },
+    "host": "{{.Host}}",
+    "basePath": "{{.BasePath}}",
+    "paths": {
+        "/healthz": {
```

Comment:
typo or wanted?

Reply:
Intended. I thinks it's being referred to as a z-page. The idea is to prevent collision with "real" endpoints. Especially given the nature of our project "/health" might be used at some point.

I just copied this practice from the k8s docs sometime.